modules/speech-to-text-impl/src/main/java/org/opencastproject/speechtotext/impl/SpeechToTextServiceImpl.java

```diff
@@ -140,8 +140,13 @@ protected String process(Job job) throws Exception {
     List<String> arguments = job.getArguments();
     String language = arguments.get(1);
     URI mediaFile = new URI(arguments.get(0));
-    Boolean translate = Boolean.parseBoolean(arguments.get(2));
-
+    Boolean translate = false;
+    try {
+      translate = Boolean.parseBoolean(arguments.get(2));
+    } catch (NumberFormatException e) {
+      throw new SpeechToTextServiceException("The translate parameter must be a boolean. Declared value: \""
+              + arguments.get(2) + "\"");
+    }
```

Comment:
Okay, this kind of my bad for not catching this the first time around. This seems to be same idea used in

`private Boolean getTranslationMode(MediaPackage mediaPackage, WorkflowInstance workflowInstance)`

in the `SpeechToTextWorkflowOperationHandler`. However, this will not throw an exception for "non-boolean" strings like 

`         <configuration key="translate">archive,subtitle,-download</configuration>`

but simply parse them as false. I would like it if it threw an exception instead.

Reply:
Thanks for catching that again, It was recommended to use `NumberFormatException` for this case, still a wrong parameter can be pass through.

I will change to use the `ApacheUtils` approach, https://stackoverflow.com/a/60277643/6308057

Reply:
Well that approach don't do the same (At least detect a series of values commonly associated with true or false).

The true/false check is in the WoH java file instead of the impl java file. Should we have a double check?